skills/inferskills.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import json
+
+
+class InferSkills:
+  def __init__(self):
+    self.vals = {}
+    with open('skillsets_labels.tsv', 'r') as f:
+      for line in f:
+        items = line.split('\t')
+        k = items[1]
+        v = set(items[2].replace('\n', '').lower().split(', '))
+        self.vals[k] = v
+  
+  def render_scores(self, data, scores, keywords):
```

Comment:
Can you please add in-line comments to document the purpose of each function, and any pertinent non-obvious parts within them?